ss58/mod.ts

```diff
@@ -1,17 +1,113 @@
-// TODO: brands
-// TODO: narrow error typings
-import { instantiate } from "./mod.generated.js";
-
-// TODO: `encodeBuf` and `decodeBuf`, if useful.
-export interface Ss58 {
-  encode(prefix: number, pubKey: string): string;
-  decode(text: string): [number, string];
-}
-
-export async function Ss58(): Promise<Ss58> {
-  const instance = await instantiate();
-  return {
-    encode: instance.encode,
-    decode: instance.decode as Ss58["decode"],
-  };
-}
+import { blake2b } from "../deps/hashes.ts";
+import * as base58 from "../deps/std/encoding/base58.ts";
+import * as U from "../util/mod.ts";
+
+// SS58PRE = Uint8Array.from("SS58PRE".split("").map((c) => c.charCodeAt(0)));
```

Comment:
Is there a reason to keep this commented alternative around?

Reply:
I left it to state that the Uint8Array from the next line is the `SS58PRE` prefix.
I'll remove the code and write a better comment